students/lauraannf/Lesson04/mailroom.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Sep 24 13:06:38 2018
+
+@author: Laura.Fiorentino
+"""
+import datetime
+from pathlib import Path
+my_path = 'C:\\Users\\Laura.Fiorentino\\Documents\\MyPython\\\
+Self_Paced-Online\\students\\lauraannf\\Lesson04\\'
```

Comment:
note that you can also get current path programmatically so you don't have to hard code

Reply:
Can you explain this to me? I was having a hard time figuring out how using paths in python. 
For this I intended to name the path (so that I could it could be any folder). What would be the most efficient way to have written this?

Reply:
My initial response was to use `os.getcwd()` which gives you current working directory. This would give you the correct path if you are always running it from Lesson4 directory.

If you want to run this from any directory then you need to use this:
```
os.path.dirname(os.path.abspath(__file__)
```

Note: I tested this on unix based system; sometimes windows is a little weird on working with file paths. 

But the idea here is that you can get the path of your current file programmatically.